### ISSUE TITLE
Failed to Compile Error

### ISSUE THREAD

**Issue by mdwiltfong** [mdwiltfong/Codecademy-Chapter-Meetups#1]
Hello Everyone! 
I'm having an issue with running create-react-app templates. At the moment when i run "npm start" in my terminal I get the following error:
Failed to compile
./src/index.js 1:45
Module parse failed: Unexpected token (1:45)
File was processed with these loaders:
 * ./node_modules/@pmmmwh/react-refresh-webpack-plugin/loader/index.js
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
> $RefreshRuntime$ = require('C:/Users/Michael's Tablet/Documents/Codeacademy/random/Codecademy-Chapter-Meetups/to-do/node_modules/react-refresh/runtime.js');
| $RefreshSetup$(module.id);
|
I've attached a picture of the browser with the same error message. This is becoming quit a problem! I basically can't develop any react apps. 
TLDR-
How can I get rid of this error message?

![image](https://user-images.githubusercontent.com/76107997/111002103-d6d87900-8352-11eb-8463-c80f7a9d4a41.png)

**Comment by mdwiltfong** [comment]
Though I haven't solved this issue, I have found a work around that allows me to continue with developing my skills in React. 

1. Run npx create-react-app
2. Change react-scripts to 4.0.1 IN ONLY THE PACKAGE.JSON FILE
3. Run npm install in the terminal
4. npm start

Hopefully developing react projects with an older version of react-scripts won't be an issue :S
